pycose/keys/cosekey.py

```diff
@@ -94,6 +94,25 @@ def from_dict(cls, received: dict) -> 'CK':
 
         return key_obj
 
+    @classmethod
+    def from_cryptography_key(
+        cls,
+        ext_key,
+        optional_params: Optional[dict] = None
+    ) -> 'CK':
+        """
+        Initialize a COSE key from a cryptography key.
+
+        :param ext_key: A cryptography key.
+        :param optional_params: Optional parameters to add to the key.
+        :return: An initialized COSE Key object.
+        """
+
+        for key_type in cls._key_types.values():
```

Comment:
Iterating over `_key_types` is not wrong, but I find it a bit weird, especially considering that `_key_types` actually have three entries per key-type. On the other hand I'm not sure I can think of a much cleaner solution other than having just a list of key types? Or a separate map from cryptography types to pycose types. None of these are particularly elegant either, so maybe it's fine to keep this as is.